deploy/cloud-node-controller-role.yaml

```diff
@@ -8,6 +8,8 @@ rules:
 - apiGroups:
   - networking.gke.io
   resources:
+  - network
```

Comment:
why do you need network

Reply:
I need get for network. I could segment it out into get only and get and update for network status.

Reply:
please do

Reply:
done